README.md

```diff
@@ -68,28 +63,19 @@ pax -ppx -rf ../<pax-name>.pax
 ssh ibmuser@my.mainframe.com       
 ```
 
-### 2) install dependencies    
-```
-cd </usr/lpp/extender>/sample-node-api        
-npm install --only=prod       
-
+### 2) install component using zowe-install-component.sh script
 ```
 
-### 3) Manage lifecycle of service with core zowe components
+./<zowe-runtime-dir>/bin/zowe-install-component.sh -c <component-name> -i <zowe-instance-dir> -o <component-pax-file> -l <log-folder>
```

Comment:
-c here is not important because our package has manifest, so the installer can get this info.
-d is more worthy to mention let's say `-d <zowe-extensions-dir>`